### PR TITLE
AC Automated Test - LRAC-11059 | master

### PR DESCRIPTION
**Pull request by manuferreira**
https://issues.liferay.com/browse/LRAC-11059 (automation ticket)
https://issues.liferay.com/browse/LRAC-8988 (bug)

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfilePath.testcase [hunk 1]
```diff
@@ -137,4 +137,58 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-8988 | Automation ID: LRAC-11059 | Test Summary: Create the page name with special characters makes the correct name appear in the path part of AC pages"
+	@priority = "3"
+	test CreatePageNameWithSpecialCharactersMakeCorrectNameAppearInPath {
+		task ("Create a site and a blog and add to a page") {
+			ACUtils.addSiteAndPage();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfilePath.testcase:144]: Hey @manuferreira, I think you should update tearDown to delete the site you created so it doesn't affect other tests in sequence:

`else {
	 JSONGroup.deleteGroupByName(groupName = "Site Name");
}`


What do you think?

**Review by ViniLopes20 — COMMENTED** [review]
Hey @manuferreira, I remember you told me that the path tab wasn't loading when it was run upstream, so I think you should put the property in the tests to only run on the release branch. What do you think?

`property analytics.cloud.upstream = "false";`

**Review by joshitagaki — APPROVED** [review]
Nice! LGTM with Vini's suggestions

**Review by ViniLopes20 — APPROVED** [review]
LGTM, thanks @manuferreira 😆

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSites.path [hunk 2]
```diff
@@ -141,6 +141,11 @@
 	<td>//*[contains(@class,'table-hover')]//span[text()='${key_nameList}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SITE_PATH</td>
+	<td>//*[name()='svg']//*[@class='analytics-sankey-title text-truncated' and @y='0']//*[text()='${key_pageName}']</td>
```
[the remark this answers is not in the document]
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSites.path:146]: I'm gonna take a look :)

[the remark this answers is not in the document]
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSites.path:146]: Hey @calvinctvo I tested on UAT using our AC QA Workspace, and my property manu 3.2.2, I selected one of my pages in the list: `Blog ac title - Site Name - Liferay DXP` and I used this xpath here: `//*[name()='svg']//*[@class='analytics-sankey-title' and @y='0']//*[text()='Blog ac title - Site Name - Liferay DXP']` and it's not working ): maybe I did something different than you did?

**Review by calvinctvo — APPROVED** [review]
Thanks @manuferreira , the code LGTM

**Comment by ViniLopes20** [comment]
Hi team, Thanks for reviewing!

I'm forwarding it!

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**